File: admin-find.py
#!/usr/bin/env python
import sys

### DIARIOSEC GROUP ###
### B3RG & IRONBITS ###
### www.Ironbits.me ###

try:
    import requests
except Exception:
    print "You don't have 'requests' library installed. Please run 'easy_install requests' as root to install it."
    sys.exit(1)

url = sys.argv[1]
if url.count("http://") <> 1:
    url = "http://"+url

if len(sys.argv) == 3:
    print ""
    print "====== Requesting to %s ======" %(url)
    print ""
    f = open(sys.argv[2])
    for path in f:
        request = requests.get(url+path)
        if request.status_code == 200:
            print path
    print ""
    print "====== End requesting to %s ======" %(url)
    print ""
else:
    print "Usage: path_request.py "

sys.exit(0)

File: admin-find.pl
#!/usr/bin/perl

### DIARIOSEC GROUP ###
## B3RG & IRONBITS ##
### www.Ironbits.me ###

use LWP::UserAgent;

$ua = LWP::UserAgent->new;


if ($#ARGV + 1 != 2){
	print "Usage: scylla_f1nd3r <url> <path>\n";
	exit;
}

if ($ARGV[0] =~ m/^http/) { $url = $ARGV[0]; } else { $url = "http://".$ARGV[0]; }

$file = $ARGV[1];

if (!open(FILE, $file)){
	print "Invalid file!\n";
	exit;
}

print "\n====== Requesting to $url ========\n";

while (<FILE>){
	if ($_ =~ m/^\//){
		$request = HTTP::Request->new(GET => "$url$_");
		$response = $ua->request($request);
		if ($response->is_success){
			print $response->status_line." : $_";
		}
	}
}

print "\n\n";
close(FILE);
exit;
